thavalon-webapp/src/components/gameComponents/roleInformation.tsx

```diff
@@ -0,0 +1,116 @@
+
+    /**
+     * Creates the JSX Element for representing the role information
+     * section of a game page.
+     * @param roleInfo The role info received in a snapshot message.
+     */
+    function createRoleInfoElement(roleInfo: RoleInfo): JSX.Element {
+        return (
+            <div id="playerInfo">
+                <h1 className="gameSectionHeader">Player Info</h1>
+                You are <span className={roleInfo.team.toLowerCase()}>{roleInfo.role}: {roleInfo.team.toUpperCase()}</span>
+                <br />
+                {roleInfo.description && <>
+                    <span className="roleDescription">{roleInfo.description}</span>
+                    <br />
+                </>}
+
+                {roleInfo.abilities && <>
+                    <span className="abilities">{roleInfo.abilities}</span>
+                    <br />
+                </>}
+
+                {roleInfo.seenPlayers.length > 0 && <>
+                    <span className="seenPlayers">You see {roleInfo.seenPlayers.join(", ")}</span>
+                    <br />
+                </>}
+
+                {roleInfo.team == "Evil" && roleInfo.teamMembers.length > 0 && <>
+                    <span className="teamMembers">You see {roleInfo.teamMembers.join(", ")} as evil</span>
+                    <br />
+                </>}
+
+                {roleInfo.otherInfo && <>
+                    <span className="otherInfo">{roleInfo.otherInfo}</span>
+                    <br />
+                </>}
+
+                {roleInfo.assassinatable && <>
+                    <span className="assassinatable">You are assassinatable!</span>
+                    <br />
+                </>}
+
+                {roleInfo.isAssassin && <>
+                    <span className="assassin">You are the assassin!</span>
+                    <br />
+                    {roleInfo.priorityTarget !== "None" && <span className="priorityTarget">{roleInfo.priorityTarget} is the priority target!</span>}
+                    {roleInfo.priorityTarget === "None" && <span className="priorityTarget">There is no priority target.</span>}
+                    <br />
+                </>}
+            </div>
+        )
+    }
+
+    /**
+     * Handles any lobby messages that come from the server. If the message type
+     * is a PlayerList change, the playerList is updated accordingly.
+     * @param message An incoming message from the server
+     */
+    function handleGameMessage(message: InboundMessage): void {
+        switch (message.messageType) {
+            case InboundMessageType.Snapshot: {
+                const snapshot = message.data as Snapshot;
+                console.log("SETTING ROLE INFO ELEMENT!");
+                setRoleInfoElement(createRoleInfoElement(snapshot.roleInfo));
+                break;
+            }
+            default: {
+                console.log("Received unsupported message type: " + message.messageType);
+                break;
```

Comment:
I don't think we should have this default case. The vast majority of messages will be "unsupported" by this component.

Reply:
We still have this default here. I think we can just get rid of it entirely if it's only a break.